xdsl/dialects/builtin.py

```diff
@@ -1643,21 +1643,30 @@ def __init__(
         self,
         ops: list[Operation] | Region,
         attributes: Mapping[str, Attribute] | None = None,
+        properties: dict[str, Attribute] | None = None,
```

Comment:
Ad properties are not extensible, and would only contain the sym_name, I would just pass a `StringAttr | None` instead, and create the properties dictionary in the __init__

Reply:
Makes sense. Done, thanks.